src/main.ts

```diff
@@ -1,5 +1,9 @@
 import Vue from 'vue'
 import App from './App.vue'
+
+declare function n(a: string): string;
+declare function t(a: string): string;
```

Comment:
* `t` method has more than one parameter
* probably we do not need this. Since in `WebDavUrl.vue` the `t` mehod in  defineComponent.method.t <- is missing

Reply:
Fixed by using imports from `@nextcloud/l10n`